backend/src/v5/processors/teamspaces/projects/models/drawings/calibrations.js

```diff
@@ -113,11 +115,13 @@ Calibrations.getCalibrationStatusForAllRevs = async (teamspace, project, drawing
 	return results;
 };
 
-Calibrations.addCalibration = async (teamspace, project, drawing, revision, createdBy, calibration, drawingData) => {
+Calibrations.addCalibration = async (teamspace, project, drawing, revision, createdBy, calibration) => {
 	const existingCalibration = await getCalibration(teamspace, project, drawing, revision, { _id: 1 });
 
-	await addCalibration(teamspace, project, drawing, revision, createdBy, calibration);
-	await updateModelSettings(teamspace, project, drawing, { calibration: drawingData });
+	await addCalibration(teamspace, project, drawing, revision, createdBy,
```

Comment:
I think I said this in the middleware 🙈 😆, use the spread operator to avoid overwriting them with undefined... And I think you still need units for the horizontal ones?

```suggestion
    const { verticalRange, units, ...calibrationData} = calibration
    await addCalibration(teamspace, project, drawing, revision, createdBy, {...calibrationData, units})
    await updateModelSettings(teamspace, project, drawing,
		{ calibration: { verticalRange, units} })
```

Reply:
unless i'm mistaken we can do this in parallel so should wrap it around a promise.all